src/main/webapp/WEB-INF/tags/base.tag

```diff
@@ -1,4 +1,7 @@
 <%@tag description="Main base template for chat app"%>
+<%@ attribute name="bodyContent" required="true" rtexprvalue="true" %>
+<%@ attribute name="headContent" rtexprvalue="true" %>
+<%@ attribute name="title" rtexprvalue="true" %>
```

Comment:
bodyContent is basically your main HTML for your page
headContent is any HTML you want to add to the `head` tag
title is an optional attribute that will set the tab's title in the browser. Defaults to "EastCode Chat"